### PR TITLE
[Convert method references ]Lps 163778

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-163778

This pull removed the RemoteMVCPortlet  according to the comment in https://github.com/brianchandotcom/liferay-portal/pull/124198#issuecomment-1270315848, then convert the method references to field references in MarketplaceStorePortlet.

/cc @dantewang @tinatian

### CODE REVIEW

**Comment by slnn** [comment]
Hi @zhangyan0701 

Could you help to scan test results when they are finished?

Thanks
Lily

File: modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java [hunk 1]
```diff
@@ -75,46 +73,6 @@
  */
 public class RemoteMVCPortlet extends MVCPortlet {
 
-	public void authorize(
```
Comment (tinatian) [inline modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java:78]: @slnn @dantewang I do not think you can remove this, cause there is usage in jsp like
`<liferay-portlet:actionURL name="authorize" var="authorizeURL">
		<portlet:param name="callbackURL" value="<%= callbackURL %>" />
	</liferay-portlet:actionURL>`

@vicnate5 This pull passed on ci, I am wondering if we have test related to marketplace authorization?

**Comment by tinatian** [comment]
@slnn Please see my comment and resend.

Thanks.
Tina

File: modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java (hunk 1 above)
Comment (vicnate5) [reply modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java:78]: Going to run tests at https://github.com/vicnate5/liferay-portal/pull/2174. Ideally I can find this breaking something so I know we have coverage.

Comment (vicnate5) [reply modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java:78]: Confirmed with automation tests that this breaks Marketplace. No errors logged, but the Marketplace app does not render.

**Comment by slnn** [comment]
Hi @tinatian

Resend in https://github.com/liferay-core-infra/liferay-portal/pull/2279, I discarded this change https://github.com/liferay-core-infra/liferay-portal/pull/2000/commits/fbdede18dfa0510f1618e4b408d94fb10b3086ac in the new pull, I'm sorry that I do not notice that both authorize and deauthorize are used in jsp


Lily

**Pull request closed without merging**